pytorch_lightning/utilities/device_parser.py

```diff
@@ -181,7 +197,9 @@ def _sanitize_gpu_ids(gpus: List[int]) -> List[int]:
         MisconfigurationException:
             If machine has fewer available GPUs than requested.
     """
-    all_available_gpus = _get_all_available_gpus()
+    if not sum([int(include_cuda), int(include_mps)]) > 0:
```

Comment:
nit: Could be simpler?
```suggestion
    if sum((include_cuda, include_mps)) == 0:
```